modules/nf-core/modules/ensemblvep/build.sh

```diff
@@ -20,9 +20,9 @@ build_push() {
     docker push nfcore/vep:${VEP_TAG}.${GENOME}
 }
 
-build_push "GRCh37"    "homo_sapiens"           "104" "104.3"
-build_push "GRCh38"    "homo_sapiens"           "104" "104.3"
-build_push "GRCm38"    "mus_musculus"           "102" "104.3"
-build_push "GRCm39"    "mus_musculus"           "104" "104.3"
-build_push "CanFam3.1" "canis_lupus_familiaris" "104" "104.3"
-build_push "WBcel235"  "caenorhabditis_elegans" "104" "104.3"
+build_push "GRCh37"    "homo_sapiens"           "105" "105.0"
```

Comment:
@maxulysse should the igenomes.config be updated here as well? or separate PR?

Reply:
Oooh, good point, I forgot about it, yes, it should be done here